app/proguard-rules.pro

```diff
@@ -73,3 +73,10 @@
 -dontwarn org.conscrypt.**
 # A resource is loaded with a relative path so the package of this class must be preserved.
 -keepnames class okhttp3.internal.publicsuffix.PublicSuffixDatabase
+
+####################################################################################################
+# Glean
+####################################################################################################
+# Keep code generated from Glean Metrics
+-keep class org.mozilla.focus.GleanMetrics.** {  *; }
```

Comment:
```suggestion
```

You shouldn't need this, because we already have the [proper rules exposed](https://searchfox.org/glean/source/glean-core/android/proguard-rules-consumer.pro). See [here as well](https://searchfox.org/glean/rev/eaecafc213574108470d7a18fba15e3b928140c9/glean-core/android/build.gradle#53,59).